source/d3d8to9_device.cpp

```diff
@@ -52,29 +52,15 @@ bool LoadD3dx9()
 	}
 	else
 	{
-		// Declare vars
+		// Load dll
 		HMODULE dllHandle = NULL;
-		char d3dx9name[MAX_PATH];
-
-		// Check for different versions of d3dx9_xx.dll
-		for (int x = 99; x > 9 && dllHandle == NULL; x--)
-		{
-			// Get dll name
-			strcpy_s(d3dx9name, "d3dx9_");
-			char buffer[11];
-			_itoa_s(x, buffer, 10);
-			strcat_s(d3dx9name, buffer);
-			strcat_s(d3dx9name, ".dll");
-
-			// Load dll
-			dllHandle = LoadLibrary(d3dx9name);
-		}
+		dllHandle = LoadLibrary("d3dx9_43.dll");
```

Comment:
Unicode please? =) This is the first ever line which will be affected by multibyte/Unicode switch so while I'm not a fan of using MS macroes for multibyte/Unicode, ideally you could replace those two lines with

```c++
HMODULE dllHandle = LoadLibrary(TEXT("d3dx9_43.dll"));
```

and support both character encodings at once. Still, project should probably end up being switched to Unicode as it's the way Windows-only projects should be handled these days to be honest.